VideoLocker/res/values/strings.xml

```diff
@@ -165,6 +165,17 @@
     <string name="end_user_title">End User License Agreement</string>
 
     <!-- My Courses -->
+    <!-- Title of course dashboard screen -->
+    <string name="course_dashboard">Dashboard</string>
+    <!-- Course Handout in dashboard -->
+    <string name="course_db_handout">Handouts</string>
+    <!-- Course announcement in dashboard -->
+    <string name="course_db_announcement">Announcement</string>
```

Comment:
Sounds good. Maybe just with a comment or something so we don't leave it in by accident.